btbphylo/consistify.py

```diff
@@ -55,22 +54,56 @@ def consistify(wgs, cattle, movements):
     # subsample full datasets by common names
     wgs_consist = wgs.loc[wgs["Submission"].isin(consist_samples)].copy()
     cattle_consist = cattle[cattle.CVLRef.isin(consist_samples)].copy()
-    movements_consist = movements[movements.SampleName.isin(consist_samples)].copy()
-    # removes NaNs from Stay_Length column to avoid error in ViewBovine
-    movements_consist = movements_consist[movements_consist['Stay_Length'].notna()]
+    movement_consist = movement[movement.SampleName.isin(consist_samples)].copy()
+    return wgs_consist, cattle_consist, movement_consist,\
+        missing_wgs, missing_cattle, missing_movement
+
+def clade_correction(wgs, cattle):
+    """
+        Ensures that the clade assigment in cattle csv matches the clade in
+        WGS data. Assumes wgs clade is correct and overwrites the cattle calde
+        if there is a mismatch. This feature corrects an error where the wrong 
+        clade is assigned in the MDWH.
```

Comment:
Slightly worrying that there's a mismatch, but you're right in assuming the WGS version will be correct.  From a quick assessment of the list that Cam sent, it seems to be linked to the lower quality samples, so they would be excluded from Viewbovine anyway.  Essentially, they don't have a clade in MDWH, but WGS has assigned one, despite the data being low quality

Reply:
Ok, from my memory we were experiencing some issues within the app) as a result of the clade mismatch (I think in some cases genetic related samples were coming through but with the wrong clade, so perhaps a few of the offending samples slip through above the filter thresholds. Which parameter were you looking at when deciding they are low quality?

I feel that either way it's not going to hurt correcting for this and is probably the safest /easiest thing to do , at least in the short term. Until we get to the bottom of the source of the mismatch ...